### PR TITLE
Callouts fixes

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Addresses a bug where FF browser wouldn't open the form when people hit the button.
Addresses another bug where multiple checkbox entries marked as required would force people to tick all the boxes 🤣

### CODE REVIEW

File: static/src/javascripts/projects/journalism/views/campaignForm.html [hunk 1]
```diff
@@ -58,7 +58,6 @@ <h4 class="campaign--snippet__headline"> <%= data.title %></h4>
                        value="<%=option.value%>"
                        id="<%=option.label%>"
                        name="field_<%=field.id%>"
-                    <% if (field.required == "1") { %> required <% } %>
```
Comment (JustinPinner) [inline static/src/javascripts/projects/journalism/views/campaignForm.html]: Are we saying that required fields can't apply to checkbox and radio buttons?

**Comment by regiskuckaertz** [comment]
[Callouts bugs in FF and in general](https://trello.com/c/bnjJQH5s/800-callouts-bugs-in-ff-and-in-general)

File: static/src/javascripts/projects/journalism/views/campaignForm.html (hunk 1 above)
Comment (regiskuckaertz) [reply static/src/javascripts/projects/journalism/views/campaignForm.html]: yes, esp. not when there are more than one (which is always the case). browsers aren't smart enough to know that _at least_ one in the group should be ticked, not all.

**Review by JustinPinner — APPROVED** [review]
👍 :shipit:

**Pull request merged**